<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1325</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow
{
	background-color:rgb(33,33,33);
	background-image:url( H:/Github/PythonScripts/PyQtDesigner/Watcher/bg1.jpg);
}

QWidget#centralwidget
{
	background-image:url( H:/Github/PythonScripts/PyQtDesigner/Watcher/bg1.jpg);
	background-color:rgb(33,33,33);
}

QLabel#labelMotionLogs
{
	color:rgb(224, 224, 224);
}

QLabel#labelWebTitle
{
	color:rgb(224, 224, 224);
}

QPushButton#startButton
{
font-size: 16px;
background-color: rgb(52, 140, 11);
border-radius: 15px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
box-shadow: 10px 10px rgb(255, 167, 38);
	
 }

QPushButton#startButton:hover:pressed 
{
border-style: outset;
border-radius: 15px;
border-width:1.2px;
box-shadow: 10px 10px rgb(255, 152, 0);
	
 }


QPushButton#stopButton
{
font-size: 16px;
background-color: rgb(229, 27, 27);
border-radius: 15px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
box-shadow: 10px 10px rgb(255, 167, 38);
	
	
 }

QPushButton#stopButton:hover:pressed 
{
	
border-style: outset;
border-radius: 15px;
border-width:1.2px;
box-shadow: 10px 10px rgb(255, 152, 0);
	
 }

QPushButton#saveLogsButton[flat=&quot;false&quot;]
{
font-size: 16px;
background-color: orange;
border-radius: 15px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
box-shadow: 10px 10px rgb(255, 167, 38);

}


QPushButton#saveLogsButton:hover:pressed 
{ 

border-style: outset;
border-radius: 15px;
border-width:1.2px;
box-shadow: 10px 10px rgb(255, 152, 0);
 
}







	
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidgetLogs">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>70</y>
      <width>271</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="logsCount">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>590</y>
      <width>141</width>
      <height>55</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelWebCamera">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveLogsButton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>590</y>
      <width>110</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Save Logs</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>110</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>Start </string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>590</y>
      <width>110</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWebTitle">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Webcamera Feed </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMotionLogs">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Motion Logs</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1325</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionManage_Users"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionManage_Users">
   <property name="text">
    <string>Manage Users</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionManage_Users</sender>
   <signal>triggered()</signal>
   <receiver>logsCount</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
